PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/common/AzureFormPanel.java

```diff
@@ -30,4 +30,6 @@
     // CHECKSTYLE IGNORE check FOR NEXT 1 LINES
     default void $$$setupUI$$$() {
     }
+
+    void setDeploymentVisible(boolean visible);
```

Comment:
should `setDeploymentVisible` be an method of interface, will all azure form panel have deployment component?